projects/core/src/occ/adapters/cms/occ-cms-component.adapter.spec.ts

```diff
@@ -241,69 +189,92 @@ describe('OccCmsComponentAdapter', () => {
         }
       );
 
-      expect(testRequest.request.responseType).toEqual('json');
       expect(testRequest.cancelled).toBeFalsy();
+      expect(testRequest.request.responseType).toEqual('json');
       testRequest.flush(componentList);
     });
 
-    it('should use a post request when get fails to get a list of cms component data with pagination parameters', () => {
-      spyOn(endpointsService, 'getUrl').and.returnValues(
-        endpoint +
-          `/components?componentIds=${ids.toString()}&productCode=${
-            context.id
-          }`,
-        endpoint + `/components?productCode=${context.id}`
+    it('should use normalizer', () => {
+      spyOn(endpointsService, 'getUrl').and.returnValue(
+        endpoint + '/components'
       );
 
-      service.findComponentsByIds(ids, context).subscribe(result => {
+      service.findComponentsByIds(ids, context).subscribe();
```

Comment:
Is it possible to refactor these tests to avoid code repetition?

Reply:
same as above